### PR TITLE
Ignore the submissions that have 0 score.

### PR DESCRIPTION
**Pull request by muzaffaryousaf**
Ignore the submissions that have 0 score.

https://openedx.atlassian.net/browse/TNL-1125

### CODE REVIEW

File: submissions/api.py [hunk 1]
```diff
@@ -467,8 +469,8 @@ def get_top_submissions(course_id, item_id, item_type, number_of_top_scores, use
             for score in scores
         ]
 
-    # Always store the retrieved list in the cache
-    cache.set(cache_key, top_submissions, TOP_SUBMISSIONS_CACHE_TIMEOUT)
+        # Always store the retrieved list in the cache
+        cache.set(cache_key, top_submissions, TOP_SUBMISSIONS_CACHE_TIMEOUT)
```
Comment (muzaffaryousaf) [inline submissions/api.py]: @stephensanchez does this make sense to set the cache only after we hit the database. Please let me know what do you think ? 
Thanks.

**Comment by muzaffaryousaf** [comment]
@stephensanchez can you please review the code for this PR.
Thanks!

File: submissions/api.py (hunk 1 above)
Comment (stephensanchez) [inline submissions/api.py]: It makes sense to me to set it after the database call. We check on line 440 if the top submissions are cached yet, if not, we query, then set the results in the cache. What did you have in mind?

Comment (stephensanchez) [inline submissions/api.py]: Oh I think I understand now, you're referring to whether your change is valid. It seems reasonable to me.

**Comment by stephensanchez** [comment]
:+1:

File: submissions/api.py (hunk 1 above)
Comment (muzaffaryousaf) [inline submissions/api.py]: Yeah, Before we were setting the cache in any case either we hit the db or cache. Now, I've changed this and only setting the cache when we hit the db.

Comment (stephensanchez) [inline submissions/api.py]: Makes sense!

**Pull request merged**